### PR TITLE
chore: update Microsoft.UI.Xaml and improve Progressring and PipsPager

### PR DESCRIPTION
**Pull request by Arieldelossantos**
﻿GitHub Issue: https://github.com/unoplatform/Uno.Themes/issues/1142

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## Description

update Microsoft.UI.Xaml and improve Progressring and PipsPager

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [x] Tested WASM
- [ ] Tested MacOS
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Comment by Arieldelossantos** [comment]
@kazo0 all the Platforms are showing ProgressRing but Android nor iOS, but i've tested this on Master too, and it's not shown neither on those two platform, so it is not related to this change but i'm wondering whether it's a known issue or not...
 with PipsPager is all good.

**Review by kazo0 — APPROVED** [review]

**Review by Xiaoy312 — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

File: src/library/Uno.Material/Styles/Controls/v1/ProgressRing.xaml [hunk 1]
```diff
@@ -1,54 +1,60 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```
[the remark this answers is not in the document]
Comment (Arieldelossantos) [reply src/library/Uno.Material/Styles/Controls/v1/ProgressRing.xaml:1]: done

**Review by Xiaoy312 — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

File: src/samples/UWP/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj [hunk 2]
```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
     <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.6.0-prerelease.210113001</Version>
+      <Version>2.8.5</Version>
```
Comment (pictos) [inline src/samples/UWP/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj:16]: This version isn't the same as the others, is it intended?

Comment (agneszitte) [reply src/samples/UWP/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj:16]: good catch @pictos, it should be 2.7.1 for all csproj @Arieldelossantos

Comment (agneszitte) [reply src/samples/UWP/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj:16]: It was intended like that @pictos 
(See details here : https://github.com/unoplatform/Uno.Themes/issues/1142#issue-1843908717)

Comment (Arieldelossantos) [reply src/samples/UWP/Uno.Themes.Samples.UWP/Uno.Themes.Samples.UWP.csproj:16]: yes, it's intended to use 2.8.5 on Sample UWP

**Comment by agneszitte** [comment]
[@Mergifyio](https://github.com/Mergifyio) backport legacy/2x

**Pull request merged**